airspeed/__init__.py

```diff
@@ -483,7 +483,7 @@ def calculate(self, namespace, loader):
 
 
 class InterpolatedStringLiteral(StringLiteral):
-    STRING = re.compile(r'"((?:\\"|[^"])*(?:""[^"]*)*)"(.*)', re.S)
+    STRING = re.compile(r'"((?:\\"|[^"]|"")*)"(.*)', re.S)
```

Comment:
i'd think the one I had would be a tid bit more flexible. But we can talk about it